badge-maker/lib/badge-renderers.js

```diff
@@ -728,17 +697,12 @@ function forTheBadge({
       ${renderMessageText()}
     </g>`
   )
-
-  if (minify) {
-    return stripXmlWhitespace(badge)
-  }
```

Comment:
Could this change potentially have any behavioral impacts? Or was `minify` always truthy for all badge styles previously?

Reply:
No, I don't see how. `badge-renderers.js` is only used in `make-badge.js`, and in the only spot where the function is invoked it gets `minify: true`.

Reply:
If I ever have to read the SVG outputs by eye, I usually flip `minify` off. How do you normally deal with that?

There was also a point where we were using it to make the snapshot tests more readable https://github.com/badges/shields/pull/4459#discussion_r362106126 but then we switched to minified snapshots later, although I notice you've taken another approach to that elsewhere which should be less sensitive to whitespace changes, so hopefully best of both worlds there.. :+1:

Reply:
Ah, interesting. My workflow for that hasn't been great. Usually I save out the svg's from the browser and then open them in VS Code and format them.

I'd be happy to add an env var that turns off the minification, if you think that would be useful.

The formatted snapshots seem to be super nice, though! I imagine that may end up being sufficient.

Reply:
Nah. Thinking about it,

```diff
-  return stripXmlWhitespace(
+  return (
```

isn't exactly much more effort than

```diff
-    minify: true,
+    minify: false,
```

on the odd occasion we want that - Its not that common.

Reply:
I didn't make it down to `make-badge.js` last night, would've been good to do so beforehand as I would've seen `minify` was always true :laughing: